package.json

```diff
@@ -11,10 +11,16 @@
     "url": "git+https://github.com/ngParty/Angular1-scaffold.git"
   },
   "scripts": {
-    "typings": "typings install",
-    "postinstall": "npm run typings",
-    "start": "webpack-dev-server --port $npm_package_config_port --hot --inline --progress --profile --colors --watch --display-error-details --display-cached --content-base src/",
-    "build": "NODE_ENV=production webpack --config webpack-build.config.js"
+    "rimraf": "rimraf",
+    "typings": "typings",
+    "postinstall": "npm run typings -- install",
+    "start": "npm run server",
```

Comment:
I made naming up after thinking about the future a bit. You are right about angular1, but we should be heading towards angular2 upgrade and then what? Different scritps naming? If you don't agree please create issue for discussion how scripts should be named. Thanks